CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/MergeNodesPluginNGTest.java

```diff
@@ -0,0 +1,274 @@
+import au.gov.asd.tac.constellation.plugins.PluginException;
+import au.gov.asd.tac.constellation.plugins.PluginExecution;
+import au.gov.asd.tac.constellation.plugins.PluginInteraction;
+import au.gov.asd.tac.constellation.plugins.parameters.PluginParameter;
+import au.gov.asd.tac.constellation.plugins.parameters.PluginParameterController;
+import au.gov.asd.tac.constellation.plugins.parameters.PluginParameters;
+import au.gov.asd.tac.constellation.plugins.parameters.types.BooleanParameterType.BooleanParameterValue;
+import au.gov.asd.tac.constellation.plugins.parameters.types.IntegerParameterType.IntegerParameterValue;
+import au.gov.asd.tac.constellation.plugins.parameters.types.SingleChoiceParameterType.SingleChoiceParameterValue;
+import au.gov.asd.tac.constellation.views.dataaccess.DataAccessPluginCoreType;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author formalhaunt
+ */
+public class MergeNodesPluginNGTest {
+    
+    private MergeNodesPlugin mergeNodesPlugin;
+    
+    public MergeNodesPluginNGTest() {
+    }
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+    }
+
+    @BeforeMethod
+    public void setUpMethod() throws Exception {
+        mergeNodesPlugin = new MergeNodesPlugin();
+    }
+
+    @AfterMethod
+    public void tearDownMethod() throws Exception {
+    }
+    
+    @Test
+    public void getPosition() {
+        assertEquals(mergeNodesPlugin.getPosition(), 100);
+    }
+    
+    @Test
+    public void getType() {
+        assertEquals(mergeNodesPlugin.getType(), DataAccessPluginCoreType.CLEAN);
+    }
+    
+    @Test
+    public void getDescription() {
+        assertEquals(mergeNodesPlugin.getDescription(), "Merge nodes in your graph together");
+    }
+    
+    @Test
+    public void createParameters() {
+        final PluginParameters actual = mergeNodesPlugin.createParameters();
+        
+        final Map<String, PluginParameter<?>> parameters = actual.getParameters();
+        
+        assertEquals(parameters.keySet(), Set.of(
+                "MergeNodesPlugin.merge_type",
+                "MergeNodesPlugin.threshold",
+                "MergeNodesPlugin.merger",
+                "MergeNodesPlugin.lead",
+                "MergeNodesPlugin.selected"));
```

Comment:
I think it would be better to use the constant strings, e.g. `MergeNodePlugin.MERGE_TYPE_PARAMETER_ID`